tools/scripts/format_build_files.sh

```diff
@@ -4,14 +4,21 @@ set -e
 
 cd "$(dirname "$0")/../.."
 
+
 if [ "$1" == "-t" ]; then
-  if ! bazel run //test/lint/buildifier:lint &> /dev/null; then
-    echo -ne "\\e[1;31m"
-    echo "☢️☢️  Some bazel files need to be reformatted! ☢️☢️"
-    echo "$OUTPUT"
-    echo -ne "\\e[0m"
-    echo -e "✨✨ Run \\e[97;1;42m ./tools/scripts/format_build_files.sh\\e[0m to fix them up.  ✨✨"
-    bazel run //test/lint/buildifier:diff
+
+  # quieter bazel output
+  bazel_args=(
+    "--curses=no"
+    "--ui_event_filters=-info,-stdout,-stderr"
+    "--noshow_progress"
+  )
```

Comment:
Don't know if you care but some of these are duplicated with `.buildkite/tools/setup-bazel.sh`, and also some aren't there.

(e.g., maybe we want those caching ones?)

just fyi, not blocking.

Reply:
Oh nice, I'll remove `--curses=no` 👍